src/api/initialDataTypes.js

```diff
@@ -78,6 +78,13 @@ export type InitialDataRealm = {|
   realm_uri: string,
   realm_video_chat_provider: string,
   realm_waiting_period_threshold: number,
+  /**
+   * Added in server version 2.2, feature level 1. Older servers
+   * don't send this.
```

Comment:
nit: "Older servers don't send this" is redundant -- "Added in ..." covers that information